hdrh/histogram.py

```diff
@@ -597,12 +597,21 @@ def get_int_to_double_conversion_ratio(self):
     def output_percentile_distribution(self,
                                        out_file,
                                        output_value_unit_scaling_ratio,
-                                       ticks_per_half_distance=5):
-        out_file.write(b'%12s %14s %10s %14s\n\n' %
-                       (b'Value', b'Percentile', b'TotalCount', b'1/(1-Percentile)'))
+                                       ticks_per_half_distance=5,
+                                       use_csv=False):
+        if use_csv:
+            out_file.write(b'"Value","Percentile","TotalCount","1/(1-Percentile)"\n')
+        else:
+            out_file.write(b'%12s %14s %10s %14s\n\n' %
+                           (b'Value', b'Percentile', b'TotalCount', b'1/(1-Percentile)'))
+
+        if use_csv:
+            percentile_format = '%.{}f,%.12f,%d,%.2f\n'.format(self.significant_figures)
+            last_line_percentile_format = '%.{}f,%.12f,%d,Infinity\n'.format(self.significant_figures)
+        else:
+            percentile_format = '%12.{}f %2.12f %10d %14.2f\n'.format(self.significant_figures)
+            last_line_percentile_format = '%12.{}f %2.12f %10d\n'.format(self.significant_figures)
```

Comment:
this if/else could have merged with the previous if/else but works equally

Reply:
thanks for your contribution, I will apply a new tag 0.9.0 next to pick this new feature.